### ISSUE TITLE
Improve test coverage after v1.1.0

### ISSUE THREAD

**Issue by ojustino** [spacetelescope/webbpsf#591]
#584 and #588 showed decreases in test coverage that should be rectified. See their respective codecov results [here](https://app.codecov.io/gh/spacetelescope/webbpsf/pull/584) and [here](https://app.codecov.io/gh/spacetelescope/webbpsf/pull/588) for more information on the files affected. @mperrin suggested that at least some of the uncovered code (particularly new functions in `trending.py`) doesn't actually need tests and can instead take a `# pragma : no cover` comment in the appropriate places.

**Comment by mperrin** [comment]
@ojustino @BradleySappington I noticed this quite old and somewhat vague issue about test coverage. I'm not sure whether there's value in keeping open this stale issue any longer? What do you think? We could perhaps set up separate issues about improving test coverage for specific parts of the code, perhaps.

**Comment by BradleySappington** [comment]
I think targeted issues for increased coverage on certain files is probably the ideal solution.  Tangible steps that can be grabbed by anyone who is capable of understanding the file in need.  @ojustino, what are your thoughts?

**Comment by ojustino** [comment]
I agree with both of you about favoring targeted issues for matters like these. This ticket's goal was to keep us honest after hurrying in some eleventh-hour updates to v1.1.0.
